src/web/app.py

```diff
@@ -0,0 +1,44 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import os
+import openai
+from flask import Flask, render_template, request
+from pathlib import Path
+from dotenv import load_dotenv
+from azure.identity import DefaultAzureCredential
+
+default_credential = DefaultAzureCredential()
+token = default_credential.get_token("https://cognitiveservices.azure.com/.default")
+
+load_dotenv()
+
+app = Flask(__name__, static_folder=Path("static").resolve())
+
+
+@app.route("/")
+def index():
+    return render_template("index.html", prompt="What is Open AI?")
+
+
+@app.route("/", methods=["POST"])
+def completions_demo():
+    model = os.getenv("AZURE_OPENAI_MODEL", "text-davinci-003")
+    prompt = request.form["text"]
+    if not prompt:
+        prompt = "What is Open AI?"
+
+    openai.api_type = "azure_ad"
+    openai.api_key = token.token
+    openai.api_base = os.getenv("AZURE_OPENAI_ENDPOINT", None)
```

Comment:
@achandmsft had originally mentioned that we wanted this to work with both Open AI and Azure Open AI endpoints. Is that still the case?

Reply:
yes that's a key value prop. However as we will not be provisioning the OpenAI endpoint it will not need any bicep. So we ideally need a configuration that allows folk to switch between the 2. We could do this in a later iteration of this app template. cc @jongio @kristapratico

Reply:
I don't think we can use Azure AD with OpenAI endpoint...so we'd have to move back to using the key for OpenAI and Azure AD for Azure.  We'd want to introduce KV to store the secrets when we do that.